.gitignore

```diff
@@ -2,5 +2,6 @@ coverage
 lib
 /node_modules
 /test/links/test-foo/node_modules
+/npm-packages-offline-cache
```

Comment:
@dickeyxxx added in https://github.com/heroku/cli-engine/pull/148/commits/a63990707e2b8154d63d6f4d50f4f2663340e3e7